acceptance/setup/pre_suite/20_install_puppet.rb

```diff
@@ -1,6 +1,8 @@
 test_name "Install Puppet" do
-  step "Install Puppet" do
-    install_puppet
+  unless (test_config[:skip_install])
+    step "Install Puppet" do
```

Comment:
This is an example to my comment about PUPPETDB_SKIP_INSTALL, I wouldn't expect that option to skip normal provisioning tasks like this.